qmake/builddir.pri

```diff
@@ -0,0 +1,52 @@
+# Copyright 2005-2018 The Mumble Developers. All rights reserved.
+# Use of this source code is governed by a BSD-style license
+# that can be found in the LICENSE file at the root of the
+# Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+# Globally use "/debug" or "/release"
+# as build directory, if it's not
+# a shadow build, otherwise use
+# the shadowed directory.
+#
+# https://wiki.qt.io/QMake-top-level-srcdir-and-builddir
+
+# Since this file is in the "qmake" directory,
+# "$$PWD/.." is the top of the source tree.
+# absolute_path() is used to clean the path.
+TOP_SRCDIR = $$clean_path($$PWD/..)
```

Comment:
Maybe worth wrapping these in:

     greaterThan(QT_MAJOR_VERSION, 4) {
         TOP_SRCDIR = $$clean_path($$PWD/..)
     }

for both TOP_SRCDIR and TOP_BUILDDIR, if it gets rid of the warning?

You could also use the short form:

    greaterThan(QT_MAJOR_VERSION, 4): TOP_SRCDIR = $$clean_path($$PWD/..)

Reply:
I am not sure if it does get rid of the warning, though! So we need to test it. I'm fine with a throwaway commit to give it a spin with the bots.

Reply:
OK, let's do it.

Reply:
Warnings not appearing anymore